### PR TITLE
Update ERD to reflect current DB structure

### PR DESCRIPTION
**Pull request by mercedesb**
The ERD hasn't been updated in about 2.5 years, updating to reflect changes

Generated with `rake gen_erd` (note: you need to install Graphviz to run this `brew install graphviz`)

### CODE REVIEW

**Comment by simi** [comment]
Maybe we can remove it since it is not easy to keep it updated. Or we can add it to CI to compare it is at latest version.

**Comment by mercedesb** [comment]
> Maybe we can remove it since it is not easy to keep it updated. Or we can add it to CI to compare it is at latest version.

I really liked having it linked in the [contribution guidelines](https://github.com/rubygems/rubygems.org/blob/master/CONTRIBUTING.md) as someone who hasn't contributed to this repo before. It was helpful even if it was out of date. So I like the idea of adding it as a CI step.

**Comment by simi** [comment]
OK, I think you can add additional "ERD" workflow started only on `schema.rb` change (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore) generating erd and do the check using `git diff -s --exit-code doc/erd.svg`.

**Comment by simi** [comment]
Can you temporarily allow this branch on CI as well? Also CI is ubuntu based, brew is not available in there.

**Comment by mercedesb** [comment]
I looked at their docs and it seemed that [Homebrew was available](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-Readme.md#package-management)
image

And sure, I'll add this branch to the filter

**Comment by simi** [comment]
It is better to use 'apt-get install'.

**Comment by mercedesb** [comment]
Even though I allowed this branch, the new action won't run b/c the schema hasn't changed.

**Comment by simi** [comment]
You can remove the paths setting for now.

**Review by simi — APPROVED** [review]

**Comment by simi** [comment]
Looks good to me. Feel free to add back the paths and I think we can merge this. :muscle:

**Comment by mercedesb** [comment]
> Ahh, I just realized, can you add `doc/erd.svg` into CI paths?

Gladly, do you know where I do that?

File: .github/workflows/erd.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: ERD
+on:
+  pull_request:
+    paths:
+      - "db/schema.rb"
```
Comment (simi) [inline .github/workflows/erd.yml:5]: here

File: .github/workflows/erd.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  push:
+    branches:
+      - master
+    paths:
+      - "db/schema.rb"
```
Comment (simi) [inline .github/workflows/erd.yml:10]: and here :)

File: .github/workflows/erd.yml (hunk 1 above)
Comment (mercedesb) [reply .github/workflows/erd.yml:5]: ohhhh, I misunderstood your comment :) got it, yeah I'll go ahead and add it as a filter for this action

**Pull request merged**